File: src/main/java/model/equipment/armor/Body.java
package model.equipment.armor;

import model.support.allEnums.ArmorMaterial;
import model.support.allEnums.ArmorType;
import model.support.allEnums.EquipmentType;

public class Body extends Armor {
    public Body(ArmorMaterial material, double weight, int quality) {
        super(EquipmentType.ARMOR, ArmorType.BODY, material, weight, quality);
    }
}

File: src/main/java/model/equipment/armor/Body.kt
package model.equipment.armor

import model.support.enums.ArmorMaterial
import model.support.enums.ArmorType.BODY_ARMOR
import model.support.enums.BodyPart
import model.support.enums.BodyPart.*
import model.support.enums.EquipmentType.ARMOR

class Body(material: ArmorMaterial, weight: Double, quality: Int) : Armor(ARMOR, BODY_ARMOR, material, BODY, weight, quality)